`Linux build_aar_module_test` has been flaky consistently

<!-- meta-tags: To be used by the automation script only, DO NOT MODIFY.
{
  "name": "Linux build_aar_module_test"
}
-->

Test got super flaky after https://github.com/flutter/flutter/commit/606e91ad63d95f881be0c8035d6e3a8e4c66987e. /cc @blasten 

![Screen Shot 2022-04-21 at 10 50 25 AM](https://user-images.githubusercontent.com/54558023/164520704-7890d3ba-f21e-4986-a2c7-7448c5a21e1d.png)

Flaky build examples:
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20build_aar_module_test/6414/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20build_aar_module_test/6413/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20build_aar_module_test/6410/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20build_aar_module_test/6409/overview

A snippet of the flake errors:
```
     #############################################################################
[build_aar_module_test] [STDOUT] stdout:               WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING
[build_aar_module_test] [STDOUT] stdout: 
[build_aar_module_test] [STDOUT] stdout:               Debug level logging will leak security sensitive information!
[build_aar_module_test] [STDOUT] stdout: 
[build_aar_module_test] [STDOUT] stdout:               https://docs.gradle.org/7.4/userguide/logging.html#sec:debug_security
[build_aar_module_test] [STDOUT] stdout:            #############################################################################
[build_aar_module_test] [STDOUT] stdout: 
[build_aar_module_test] [STDOUT] stdout: 
[build_aar_module_test] [STDOUT] stderr: [ +758 ms] 2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':webcrypto'.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.955+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all files for configuration ':webcrypto:classpath'.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.41.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.lint:lint-gradle-api:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:gradle-api:3.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > androidx.databinding:databinding-compiler-common:3.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools:sdk-common:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools:common:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools.build:manifest-merger:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools.analytics-library:tracker:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools:sdklib:26.5.0 > com.android.tools:repository:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :webcrypto > com.android.tools.build:gradle:3.5.0 > com.android.tools.build:builder:3.5.0 > com.android.tools:sdk-common:26.5.0 > com.android.tools.analytics-library:shared:26.5.0
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.956+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]       > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.41.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.957+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          > Could not get resource 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.41/kotlin-stdlib-jdk8-1.3.41.pom'.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.957+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]             > Could not GET 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.41/kotlin-stdlib-jdk8-1.3.41.pom'.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.957+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]                > peer not authenticated
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.957+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not resolve org.ow2.asm:asm-commons:6.0.
[build_aar_module_test] [STDOUT] stderr:            2022-04-21T16:40:05.957+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:
```

--- a/.ci.yaml
+++ b/.ci.yaml
@@ -205,32 +205,6 @@ targets:
       validation_name: Analyze
     scheduler: luci
 
-  - name: Linux build_aar_module_test
-    recipe: devicelab/devicelab_drone
-    timeout: 60
-    properties:
-      add_recipes_cq: "true"
-      caches: >-
-        [
-          {"name":"gradle","path":"gradle"},
-          {"name": "openjdk_11", "path": "java"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:96.2"},
-          {"dependency": "open_jdk", "version": "11"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: build_aar_module_test
-    scheduler: luci
-    runIf:
-      - dev/**
-      - packages/flutter_tools/**
-      - bin/**
-      - .ci.yaml
-
   - name: Linux build_tests_1_2
     recipe: flutter/flutter_drone
     timeout: 60
@@ -2445,31 +2419,6 @@ targets:
         ["devicelab","android","linux"]
       task_name: android_lifecycles_test
 
-  - name: Mac build_aar_module_test
-    recipe: devicelab/devicelab_drone
-    timeout: 60
-    properties:
-      add_recipes_cq: "true"
-      caches: >-
-        [
-          {"name":"gradle", "path":"gradle"},
-          {"name": "openjdk_11", "path": "java"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "open_jdk", "version": "11"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: build_aar_module_test
-    runIf:
-      - dev/**
-      - packages/flutter_tools/**
-      - bin/**
-      - .ci.yaml
-    scheduler: luci
-
   - name: Mac build_ios_framework_module_test
     recipe: devicelab/devicelab_drone
     timeout: 60
@@ -3799,33 +3748,6 @@ targets:
       - bin/**
       - .ci.yaml
     scheduler: luci
-
-  - name: Windows build_aar_module_test
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/102226
-    recipe: devicelab/devicelab_drone
-    timeout: 60
-    properties:
-      add_recipes_cq: "true"
-      caches: >-
-        [
-          {"name":"gradle","path":"gradle"},
-          {"name": "openjdk_11", "path": "java"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:96.2"},
-          {"dependency": "open_jdk", "version": "11"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: build_aar_module_test
-    scheduler: luci
-    runIf:
-      - dev/**
-      - packages/flutter_tools/**
-      - bin/**
-      - .ci.yaml
 
   - name: Windows build_tests_1_3
     recipe: flutter/flutter_drone

